File: Problem-9/OddEven.c
// WAP to find whether a number is Odd or Even without using a % operator.

#include<stdio.h>

void main(){
    int n;

    printf("Enter n : ");
    scanf("%d", &n);

    if((n / 2) * 2 == n){
        printf("%d is Even !!", n);
    }
    else{
        printf("%d is Odd !!", n);
    }
}

File: Problem-9/OddEven.java
// WAP to find whether a number is Odd or Even without using a % operator.

import java.util.*;

public class OddEven{
    public static void main(String [] args){

        Scanner sc = new Scanner(System.in);

        System.out.println("Enter n : ");
        int n = sc.nextInt();

        if((n / 2) * 2 == n){
            System.out.println(n + " is Even !!");
        }
        else{
            System.out.println(n + " is Odd !!");
        }
    }
}